src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import { useContext } from 'react';
+import classnames from 'classnames';
+
+import { TodoContext } from '../../context/TodoContext';
+import { FilterContext } from '../../context/FilterContext';
+import { Status } from '../../interface/Status';
+
+export const TodoFooter = () => {
+  const { todos, setTodos } = useContext(TodoContext);
+  const { selectedFilter, setSelectedFilter } = useContext(FilterContext);
+
+  const clearCompleted = () => {
+    const activeTodos = todos.filter(({ completed }) => !completed);
+
+    setTodos(activeTodos);
+  };
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${todos.filter(({ completed }) => !completed).length} items left`}
```

Comment:
It is better to perform such calculations in separate variables and use the finished result.
It is also worth considering that if there is 1 element, then there will be a set: "1 items".